lib/firefox/webdriver/firefox.js

```diff
@@ -73,6 +73,14 @@ class Firefox {
    * Before each URL/test runs.
    */
   async beforeEachURL(runner) {
+    await runner.runPrivilegedScript(`
```

Comment:
Here you can just return the promise instead of await. I'll fix that when I merged :)